src/meta/src/rpc/ddl_controller.rs

```diff
@@ -1827,8 +1821,8 @@ impl DdlController {
         // 3. Build the table fragments structure that will be persisted in the stream manager, and
         // the context that contains all information needed for building the actors on the compute
         // nodes.
-        let table_fragments = TableFragments::new(
-            (dummy_table_id as u32).into(),
+        let table_fragments = StreamJobFragments::new(
```

Comment:
Class names are renamed while variables (or fields) still remain untouched. 😕

Reply:
I think renaming type is more important than renaming variables. It's so hard to change all occurances. 🥺